src/index.ts

```diff
@@ -16,9 +17,9 @@ export const createSitemapGenerator = (config: Config) => {
   const defaultConfig = getConfig(config);
 
   return {
-    sitemap: (request: Request, context: EntryContext) => {
+    sitemap: (request: Request, context: EntryContext, appContext: AppLoadContext) => {
```

Comment:
maybe the app context can be optional, we also need to update the usage documentation